src/Components/FieldTrip/FieldTripDetails.js

```diff
@@ -16,31 +10,33 @@ import StudentsReadOnlyTable from "./StudentsReadOnlyTable";
 import ChaperonesTable from "./ChaperonesTable";
 import "./FieldTripDetails.css";
 
-let perPage;
+const FieldTripDetails = ({ match }) => {
+  let perPage;
```

Comment:
@Seun-O `let perPage` needs to be moved outside above the FieldTripDetails component or else it'll be undefined on each re-render